unlock-app/next.config.js

```diff
@@ -3,7 +3,7 @@
 const fs = require('fs')
 const { join } = require('path')
 const { promisify } = require('util')
-const withSourceMaps = require('@zeit/next-source-maps')
```

Comment:
can the two extensions co-exist? Or does withTypescript auto-export source maps? If the latter, a comment above would be useful

Reply:
Added a comment just after your question--the TypeScript compiler actually emits sourcemaps, so the next extension shouldn't be necessary.